src/styles/bloks/header.scss

```diff
@@ -0,0 +1,61 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    margin: 10px 0 10px 50px;
+    cursor: pointer;
+  }
+
+  &__nav {
+    display: flex;
+
+    &-list {
```

Comment:
if it's an element of `nav` then whole nav block should be moved to a separate file
and there should be `&__list`